K2Bridge/ElasticQueryTranslator.cs

```diff
@@ -83,7 +83,7 @@ public QueryData TranslateQuery(string header, string query)
                                 elasticSearchDsl.HighlightText.Add("*", queryStringClause.Phrase);
                                 break;
                             case MatchPhraseClause matchPhraseClause:
-                                elasticSearchDsl.HighlightText.Add(matchPhraseClause.FieldName, matchPhraseClause.Phrase);
+                                elasticSearchDsl.HighlightText.Add(matchPhraseClause.FieldName, matchPhraseClause.Phrase.ToString()); // TODO - fix this when highlighting is fixed
```

Comment:
what is this todo? I am not aware of any issue with the highlighting

Reply:
Do you know how can I actually get to this line?

When adding a match clause in the UI it's always under elasticSearchDsl.Query.Bool.Filter

Reply:
try the filters